tests/integration/test_data_cli.py

```diff
@@ -701,9 +702,31 @@ def test_search_get_id_not_found(invoke, search_id):
     assert 'Error: {"message": "Error message"}\n' == result.output
 
 
+@respx.mock
+def test_search_delete(invoke, search_id, search_result):
```

Comment:
ok this is just a style thing, but I love to use pytest's `-k` option to select a test to run. That option just matches the beginning of a test name so `-k test_search_delete` would run this test as well as the next test. I tend to work around that by adding a unique identifier to the end of the name - like `test_search_delete_success`. Bonus of a little more descriptive name!

Reply:
You got it!